test/MSVS/vs-10.0-exec.py

```diff
@@ -90,6 +90,11 @@
 
 test.run(chdir='sub dir', arguments='.')
 
+for filename in ('foo.exe', 'foo.obj', '.sconsign.dblite'):
```

Comment:
Can you move this into `TestSConsMSVS`? rather than copy/pasting it into all the files?

Reply:
Sure.

Reply:
Should it be a function that takes a list of files?

Something like:
```
test.remove_files(['foo.exe', 'foo.obj', 'sconsign.dblite'])
```

Reply:
Does `scons -c` not do this?

Reply:
Pretty sure it also removes the generated msvs project.

Reply:
I'm not sure moving it to TestSConsMSVS gets much shorter as the sub directory ('sub dir') needs to be either passed separately or  joined with filenames in argument list.

Reply:
scons -c should remove all the built files.. maybe not the subdir which would be empty..  can you give that a try?

Reply:
Would the following be acceptable?

TestSConsMSVS.py
```
    def unlink_files(self, filedir, filenames):
        for filename in filenames:
            filepath = self.workpath(filedir, filename)
            if os.path.exists(filepath):
                self.unlink(filepath)
```

test scripts:
```
test.unlink_files('sub dir', ['foo.exe', 'foo.obj', '.sconsign.dblite'])
```

Reply:
does scons -c not remove the files in question?

Reply:
I'm pretty sure that scons -c removes the generated msvs project which needs to be built from the command-line with the msdev binary.  There will be no project to pass to the command-line after "scons -c" is run.

The sequence is:
* scons SConstruct (generates msvs project and builds program)
* delete program binaries
* `pathto\msdev.com foo.dsp` to build the program binaries via the generated workspace.

Reply:
gotcha. ok so it's not about cleaning all the build products, but rather removing those which should be built when you have msvs run the created project file. 

Ok. I'd add that method to TestCmd.py where there are currently other file handling functions, rather than TestSConsMSVS, as it could be useful for non MSVC tests.

Reply:
Changes:
1. Added rewritten `unlink_files` to TestCmd class with docstring and functionality consistent with `unlink` method.
2. Replaced all msvs executable script deletion loops with single test method call.
3. Reordered authors alphabetically in CHANGES.txt,